src/jdk.attach/share/classes/sun/tools/attach/HotSpotVirtualMachine.java

```diff
@@ -65,7 +65,7 @@ private static long pid() {
         ALLOW_ATTACH_SELF = "".equals(s) || Boolean.parseBoolean(s);
         // For now the default is false.
```

Comment:
Shall we remove the comment?  It was not obvious whether it applied to the code above, or below. 8-)